go.mod1

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/pingcap/errors v0.11.0
 	github.com/pingcap/tidb v0.0.0-20190218065808-69472bd1a6e9
 	github.com/pingcap/tipb v0.0.0-20190107072121-abbec73437b7
+	github.com/pkg/errors v0.8.1
```

Comment:
You can use github.com/pincap/errors which is forked from github.com/pkg/errors @wuudjac

Reply:
@tiancaiamao It's my mistake, fixed in [`f969710`](https://github.com/pingcap/parser/pull/220/commits/f96971034f46cad81282625d6c1d210e3a0a5650).